core/src/main/resources/application.yml

```diff
@@ -616,6 +616,14 @@ cb:
           7.2.16 - Streaming Analytics Light Duty with Apache Flink=cdp-flink-light;
           7.2.16 - Streaming Analytics Heavy Duty with Apache Flink=cdp-flink-heavy;
           7.2.16 - COD Edge Node=cdp-cod-edge-node
+  devblueprint:
+      enabled: false
+      cm:
+        defaults:
+          7.2.15: >
+            [DEV] 7.2.15 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie=cdp-data-engineering;
```

Comment:
maybe better to add the dev prefix to the right side name to mark (I don't remember it must be unique or not)

Reply:
why better? this way we can see it is for dev purposes instantly

Reply:
I mean the name uniqueness, but I think you checked it already

Reply:
ohh you mean that I should add `DEV` prefix from code?